wsgi/odyssy/announcement/views.py

```diff
@@ -0,0 +1,42 @@
+from django.core.paginator import Paginator, EmptyPage, PageNotAnInteger
+from django.shortcuts import render
+from django.db.models import Q
+
+from .models import Announcement
+import datetime
+
+# Create your views here.
+def index(request):
+    announcements = Announcement.objects.all().order_by('-initDate')
+    paginator = Paginator(announcements, 10)
+
+    page = request.GET.get('page')
+    try:
+        announcements = paginator.page(page)
+    except PageNotAnInteger:
```

Comment:
This exception catch can be removed if we give default value 1 to page on line 13..